rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PrepareTemplateVariables.scala

```diff
@@ -86,6 +86,76 @@ trait PrepareTemplateVariables {
 
 }
 
+object PrepareTemplateVariables extends Loggable {
+
+  /*
+   * utilitary method for formating list of "promisee usebundle => bundlename;"
+   */
+  def formatMethodsUsebundle(x:Seq[(String, Bundle)]): String = {
+
+    val alignWidth = if(x.size <= 0) 0 else x.map(_._1.size).max
+    x.map { case (promiser, bundle) =>
+      val escaped = ParameterEntry.escapeString(promiser)
+      //the promiser value (may) comes from user input, so we need to escape "
+      s""""${escaped}"${" "*Math.max(0, alignWidth - escaped.size)} usebundle => ${bundle.name};"""
+    }.mkString( "\n")
+  }
+
+  /*
+   * utilitary method for formating an input list
+   */
+  def formatBundleFileInputs(x: Seq[String]) = {
+    val inputs = x.distinct
+    if (inputs.isEmpty) {
+      ""
+    } else {
+      inputs.mkString("\"", s"""",\n${" "*4}"""", s"""",""")
+    }
```

Comment:
In fact you don't need the spaces

current output (with spaces)is: 

    inputs => {
      "rpmPackageInstallation/7.0/rpmPackageInstallation.cf",
          "test_3_2/1.0/test_3_2.cf",
    };